modules/rest-util/src/blaze/middleware/fhir/output.clj

```diff
@@ -45,6 +47,11 @@
   (with-open [_ (prom/timer generate-duration-seconds "xml")]
     (generate-xml* body)))
 
+(defn- generate-binary [body]
+  (log/trace "generate binary")
+  (with-open [_ (prom/timer generate-duration-seconds "binary")]
+    (fhir-spec/unform-binary body)))
```

Comment:
Instead of the unform-binary, I would just take the value of the data property and base64 decode it.